### DISCUSSION TITLE
Retour chariot dans les fichiers textes

### DISCUSSION THREAD

**Discussion by AvecUnHache** [skarab42/marv#115] [Ideas]
La possibilité de faire des retours chariots, aka des retours à la lignes dans les fichiers textes des fenêtres de timeline afin de pouvoir créer plusieurs lignes dans un même fichier !

**Comment by skarab42** [comment]
Dans la dernière version (1.4.0) tu peu ajouter une balise `<br />` pour faire un saut de ligne.

**Reply by AvecUnHache** [reply]
trop bien bravo !!
